persistent/Database/Persist/Sql/Types/Internal.hs

```diff
@@ -9,8 +9,8 @@
 module Database.Persist.Sql.Types.Internal
     ( HasPersistBackend (..)
     , IsPersistBackend (..)
-    , SqlReadBackend (unSqlReadBackend)
-    , SqlWriteBackend (unSqlWriteBackend)
+    , SqlReadBackend (..)
+    , SqlWriteBackend (..)
```

Comment:
I have no idea why the constructors were hidden. Presumably you were only supposed to use the `IsSqlBackend` instance, but that lets you construct them, so :man_shrugging: